### ISSUE TITLE
'npm start' issue when starting up the server

### ISSUE THREAD

**Issue by sali45** [gt-8133/inventory-manager#50]
This looks similar to the earlier node install issue. `npm i -g npm` worked for that, but it doesn't resolve this error:

```
Success! Cluster local already exists and is up-to-date.
To get started, execute
    
  $ graphcool init
  $ graphcool deploy

 ▸    No service with id 'cjllndxls00020107o32kr4h3' in account test@test.org.
 ▸    Please check if you are logged in to the right account.

Get in touch if you need help: https://www.graph.cool/forum
To get more detailed output, run $ export DEBUG="*"
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! server@1.0.0 start: `graphcool local up && graphcool deploy`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the server@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.'
```

**Comment by Bkucera** [comment]
fixed in https://github.com/gt-8133/inventory-manager/pull/49

**Comment by Bkucera** [comment]
can you paste the output of 
```
cd server
graphcool local up
```
?

**Comment by sali45** [comment]
```
Running local Graphcool cluster at http://localhost:60000
Running local FaaS runtime at http://localhost:60050
This may take several minutes
$ docker-compose up -d --remove-orphans

docker   local_graphcool-db_1 is up-to-date
docker   local_localfaas_1 is up-to-date
docker   local_graphcool_1 is up-to-date

Waiting for Graphcool to initialize. This can take several minutes... ✔

Success! Cluster local already exists and is up-to-date.
To get started, execute
    
  $ graphcool init
  $ graphcool deploy

```

**Comment by Bkucera** [comment]
Yup the problem is explained here, currently sharing local environments is not first class in graphcool https://github.com/prisma/graphcool-framework/issues/406

I plan on migrating us over to Prisma, which gives us access to the lower level docker config, which will solve this

**Comment by Bkucera** [comment]
fixed in #60
